jep/com.ibm.wala.cast.python.cpython/.launchers/PassingSuite.launch

```diff
@@ -29,5 +29,5 @@
     <listAttribute key="org.eclipse.jdt.launching.MODULEPATH"/>
     <stringAttribute key="org.eclipse.jdt.launching.PROJECT_ATTR" value="com.ibm.wala.cast.python.cpython"/>
     <stringAttribute key="org.eclipse.jdt.launching.SOURCE_PATH_PROVIDER" value="org.eclipse.m2e.launchconfig.sourcepathProvider"/>
-    <stringAttribute key="org.eclipse.jdt.launching.VM_ARGUMENTS" value="-ea -Djava.library.path=/Users/dolby/miniforge3/envs/walajep/lib/python3.13/site-packages/jep"/>
+    <stringAttribute key="org.eclipse.jdt.launching.VM_ARGUMENTS" value="-ea&#10;--enable-native-access=ALL-UNNAMED -Djava.library.path=${sitePackages}/jep/&#10;-Djava.util.logging.config.file=./logging.properties"/>
```

Comment:
@juliandolby Unsure if this will mess up your local environment. Pls feel free to revert.